spec.bs

```diff
@@ -1344,15 +1351,16 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
   1. [=Handle an input promise in configuration=] given |auctionConfig| and |auctionConfig|'s
     [=auction config/per buyer signals=]:
     * To parse the value |result|:
-      1. Set |auctionConfig|'s [=auction config/per buyer signals=] to a new [=ordered map=] whose
-        [=map/keys=] are [=origins=] and whose [=map/values=] are [=strings=].
-      1. [=map/For each=] |key| → |value| of |result|:
-        1. Let |buyer| be the result of [=parsing an https origin=] with |key|. If |buyer| is
-           failure, throw a {{TypeError}}.
-        1. Let |signalsString| be the result of [=serializing a JavaScript value to a JSON string=],
-          given |value|.
-        1. [=map/Set=] |auctionConfig|'s [=auction config/per buyer signals=][|buyer|] to
-          |signalsString|.
+      1. If |result| is not null:

```

Comment:
Seems we need to set |auctionConfig|'s [=auction config/per buyer signals=] to null when |result| is null

Reply:
... probably need to set it to that on all regardless of default because we set it to the promise first.

Reply:
yeah true.

Reply:
Done